pkg/gadgets/trace/open/tracer/tracer_test.go

```diff
@@ -115,6 +115,11 @@ func TestOpenTracer(t *testing.T) {
 		validateEvent   func(*testing.T, *utilstest.RunnerInfo, int, []types.Event)
 	}
 
+	normalize_cgroupid := func(e *types.Event) {
+		e.CgroupID = 0
+		e.Sdname = ""
```

Comment:
Shouldn't sdname be always empty? It is supposed to be enriched by the systemdresolver operator and here it is not being used. Did I miss something?